### ISSUE TITLE
Complemento de Pago

### ISSUE THREAD

**Issue by aajrm** [andresamayadiaz/autofactura#1]
Buen dia!

Excelente repositorio, la verdad me servio bastante, estoy haciendo algo similiar pero en PHP.

Para complementos de pago, usando la misma API tendra alguna idea de como integrar esa parte?

Gracias, saludos.

**Comment by andresamayadiaz** [comment]
Hola, de momento no lo tengo en PHP pero tengo una "semi clase" para timbrarlos. No se si estes buscando generarlos y timbrarlos o un API para autotimbrado de complementos de pago?

**Comment by aajrm** [comment]
No, quizas me confundi, digo que su proyecto lo estoy pasando a PHP, pero quisiera integrar la parte de complementos de pago usando la API de autofactura.

Ya intente varias formas, pero no me reconoce los Doctos relacionados.

**Comment by andresamayadiaz** [comment]
Ok, me podrías indicar cual es el método del API que estas utilizando, así como un REQUEST de ejemplo y lo reviso a detalle.
